### ISSUE TITLE
draft issues | feedback: "journal stopped after 2022"

### ISSUE THREAD

**Issue by biumiamy** [C2DH/journal-of-digital-history#605]
"journal looks so bad"
but how is it possible with all our many articles in the pipeline?

Actually, the Varia issue shows the following date:

![Image](https://github.com/C2DH/journal-of-digital-history/assets/47319502/a0beddf9-abad-4356-ae9d-fe5395bf8c64)

even though, as it's an open ended issue, there are articles from 2023 (but nobody saw that probably):

![Image](https://github.com/C2DH/journal-of-digital-history/assets/47319502/74deecb8-44cf-4149-963b-cca17b8182cc)

Possible solutions:
- show what's going on behind the scenes (articles pending)
- annual editorials
- milestone and article status could be helpful ([#464](https://github.com/C2DH/journal-of-digital-history/issues/464))
- regularily update [events](https://github.com/C2DH/journal-of-digital-history/wiki#how-to-update-the-events-in-the-milestone-section-of-the-homepage)
- add date in release notes
- change name from "Varia Issue n.2 - 2022" to "Issue n.2 - 2022-2024" (useful for open ended issues, indicating start date and current date)

example: how to show issues, open-ended issues or just announce a "coming soon" issue
<img width="313" alt="2024-02-08 08_30_30-Issue_Article OVP - Figma" src="https://github.com/C2DH/journal-of-digital-history/assets/47319502/621b99c3-3e5f-4c24-956d-4b91b8869a12">

if we show the article, need to decide what labels we want to display (e.g. draft, technical review, peer review, etc.)
<img width="926" alt="2024-02-08 08_30_58-Issue_Article OVP - Figma" src="https://github.com/C2DH/journal-of-digital-history/assets/47319502/f67f7a2a-2584-4561-a751-3502345bff07">

if we show the articles (with or without link to the preprint), need to provide more structure to not loose the overview. For example by adding the ToC of the issues (with or without article names):
<img width="420" alt="2024-02-08 08_31_36-Issue_Article OVP - Figma" src="https://github.com/C2DH/journal-of-digital-history/assets/47319502/10a2ed91-863e-440e-8c77-20383cf17cc2">

**Comment by lwieneke** [comment]
> change name from "Varia Issue n.2 - 2022" to "Issue n.2 - 2022-2024" (useful for open ended issues, indicating start date and current date)
To be confirmed, but I think we wanted to organise the open ended issues by year – so in that case there should be two more issues (2023 and 2024). But @inactinique probably has a better idea on that.

**Comment by eliselavy** [comment]
is_open_ended flag on Issue available on dev: http://10.240.4.179/api/issues/?is_open_ended=true

**Comment by eliselavy** [comment]
Agree about:
- remove the year

- displayed draft issue 
the first world of the call of paper. - + link call of the paper

**Comment by biumiamy** [comment]
- dont show article with title
- show only issue and description

[Figma](https://www.figma.com/file/oVgvRWOfeO7O8Z8aU7l6aV/Issue%2FArticle-OVP?type=design&node-id=809%3A2&mode=design&t=rD0NOaDEPjKs0KWQ-1)

design proposals:

![draft_issue_no1](https://github.com/C2DH/journal-of-digital-history/assets/47319502/f42d2a2d-fb82-419c-9152-af12bc33c4a3)

![draft_issue_no2](https://github.com/C2DH/journal-of-digital-history/assets/47319502/ccb5e9da-627c-4ca5-b942-28dd7e359258)

![draft_issue_no3](https://github.com/C2DH/journal-of-digital-history/assets/47319502/5e14e382-8003-46a8-a08f-8c49a34a3133)

**Comment by inactinique** [comment]
> * dont show article with title nor preprint
> 
>     * show ONLY issue and description (like a teaser)
> 
> 
> [Figma](https://www.figma.com/file/oVgvRWOfeO7O8Z8aU7l6aV/Issue%2FArticle-OVP?type=design&node-id=809%3A2&mode=design&t=rD0NOaDEPjKs0KWQ-1)
> 
> design proposals:
> 
> (1) horizontal alignment of issues with issue short description and link to CFP !

I would prefer this one, the (1). I think the 3rd really does not work and I'm note sure it's logical to put the coming issues above the title of the page (2).

Would it be possible to not announce the number of the issue? As those are really hard, for those three competing issues, to guess before we start their publication.

**Comment by inactinique** [comment]
Yes, but then the articles are very far away from the top of the page. Number 1, with titles being a bit greyish instead of black? Like when an item exists but is unavailable and you cannot click on it?

**Comment by danieleguido** [comment]
We could condense the description to create smaller issue cards. However, the primary issue with option 1 remains: the draft issue appears visually distinct from the published version

**Comment by inactinique** [comment]
> However, there is another problem I foresee: when we publish articles in Varia, what will happen then?

Note sure to understand the question. We won't announce Varia issues, It's not necessary.

**Comment by eliselavy** [comment]
> > However, there is another problem I foresee: when we publish articles in Varia, what will happen then?
> 
> Note sure to understand the question. We won't announce Varia issues, It's not necessary.

Last article published Delors 's article, ...

**Comment by danieleguido** [comment]
to reduce the distance from the top page, I suggest to collapse / open the description. We will surely have less issue titles without articles in the coming future, so the distance between the articles and top of the page will be reduced even more ![Screenshot 2024-02-08 at 15 11 02](https://github.com/C2DH/journal-of-digital-history/assets/1181642/c1372c15-4eab-45cd-8de5-7ed502479765)

**Comment by eliselavy** [comment]
In developemnt
http://10.240.4.179/en/articles

To look at logged and not logged

@danieleguido year need to be removed from filters / labels

**Comment by inactinique** [comment]
@danieleguido can you also remove the number of the issues, for the coming soon ones? Because, for instance, if three articles of the Tools issue are ready before the China articles, then the issue 3 will be the tools. If it's the opposite, then DH China will be the issue n. 3.

**Comment by eliselavy** [comment]
> @danieleguido can you also remove the number of the issues, for the coming soon ones? Because, for instance, if three articles of the Tools issue are ready before the China articles, then the issue 3 will be the tools. If it's the opposite, then DH China will be the issue n. 3.

you see the number of articles of the coming soon issues because you are connected to the admin.
otherwise you don't see it

**Comment by inactinique** [comment]
It's not what I meant.

**Comment by eliselavy** [comment]
Ok catch .
@danieleguido 
- Remove Issue number for issue in DRAFT status
<img width="278" alt="Screenshot 2024-02-09 at 08 50 34" src="https://github.com/C2DH/journal-of-digital-history/assets/35602279/de289857-99f8-4af1-90e3-bf930f7c2545">
- Remove the year in the issue filter
- Remove the year in the issue label
<img width="234" alt="Screenshot 2024-02-09 at 08 52 36" src="https://github.com/C2DH/journal-of-digital-history/assets/35602279/a25df90f-4493-4aab-8da3-a93ec8e4a896">

**Comment by biumiamy** [comment]
> It's better. Can it be even more collapsed?

How many lines? just 2?

**Comment by biumiamy** [comment]
Okay we will adjust it to sth like this
- change to 2 lines 
- and text adjustment

<img width="824" alt="2024-02-09 09_14_53-Issue_Article OVP - Figma" src="https://github.com/C2DH/journal-of-digital-history/assets/47319502/83099c72-18d9-4699-ad47-47e905a78a85">

**Comment by inactinique** [comment]
That's better and better. Just one comment: no date at all. Just "Coming soon". Because it can sound as a commitment.

**Comment by biumiamy** [comment]
We just discussed it again:

for the draft issues:
- use "coming soon" and add the month as soon as we're sure
- only 2 lines of text to save space

for the already published issues:
- add a date to know which year it belongs to
- Andreas suggested a yearly Varia issue

ToDo @biumiamy 
- how to show new article (e.g. "new" label)

**Comment by biumiamy** [comment]
add a "new" label when a new article is added (for 1 or 2 weeks)

<img width="639" alt="2024-02-12 11_33_44-Issue_Article OVP - Figma" src="https://github.com/C2DH/journal-of-digital-history/assets/47319502/b07b6e2d-d1ff-4bd0-b484-ee53e09de45a">

**Comment by biumiamy** [comment]
Another way to indicate that we do a yearly Varia and that we are doing many could be adding a filter directly

![Image](https://github.com/C2DH/journal-of-digital-history/assets/47319502/d7653200-95c7-47af-b0b9-1f05d8c82831)

**Comment by danieleguido** [comment]
see https://github.com/C2DH/journal-of-digital-history/pull/610 

**published** issues:
I've added the num of articles (and num of articles filtered if filters are enabled); removed the dates; added the "new" label and a coloured border

**draft** issues:
Only "coming soon" is visible, if there's a date I suggest to add in bold `<b></b>`in the description of the issue

 ![Screenshot 2024-02-12 at 15 27 40](https://github.com/C2DH/journal-of-digital-history/assets/1181642/6d6c2433-37ab-4daf-bb64-8463a94c9ac8)

**Comment by biumiamy** [comment]
After I checked, with only 3 articles for 2022, 3 for 2023 and 5 for 2024 (coming), I think we dont need to make annual Varia issues a thing for the moment. (But please share any thought on this)

**Comment by biumiamy** [comment]
Otherwise I think we need to move the "action" area to the same place as we have it on the articles:

<img width="956" alt="2024-02-13 10_34_34-Journal of Digital History" src="https://github.com/C2DH/journal-of-digital-history/assets/47319502/2f5a9d6b-2111-449b-a10e-bbc6a852362a">

And I would propose to change the sort by to something more like this:

<img width="805" alt="2024-02-13 10_43_19-Issue_Article OVP - Figma" src="https://github.com/C2DH/journal-of-digital-history/assets/47319502/bbb9dad1-8a48-468c-b1f3-78e4d6f17d7a">

**Comment by biumiamy** [comment]
here's a [new issue](https://github.com/C2DH/journal-of-digital-history/issues/611) for this topic

**Comment by eliselavy** [comment]
In prod https://github.com/C2DH/journal-of-digital-history/releases/tag/v4.4.2
